index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css" type="text/css">
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <title>Works and About Myself Sections</title>
+</head>
+<body>
+  <div class="page-wrapper">
+    <section class="work">
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio.png" alt=""></div>
+          <div class="primary-text">
+            <h1>Tonic</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(1).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
```

Comment:
- [ ] Kindly change the h1 tag to h2 because following semantic HTML on one page it should be only one h1 and it should be in the beginning. In this case, this should be h2 or less. Keep coding 💪